### ISSUE TITLE
Compiling individual instances instead of whole page

### ISSUE THREAD

**Issue by mayank99** [11ty/webc#27]
The `page.compile()` API seems like it's designed specifically to be used with a page that itself is written in the webc format. This means that any existing (unrelated) scripts and style tags on the page need to have `webc:keep` added to them to prevent webc from touching them.

It would be nice to be able to tell webc to only compile the files that it finds in the globs passed via `defineComponents` and ignore the rest of the page. Since webc knows about this list of components ahead of time, this should be possible. 🤔

One very common use case for this is when a global layout file is written in plain html/njk. This might have critical styles and inline scripts that should not be processed by webc.

**Comment by zachleat** [comment]
I just want to make sure you know that we very recently added Bundler Mode as of 0.4 and disabled it by default (though it’s enabled by default in the Eleventy plugin). Did you upgrade to 0.4?

More info at https://github.com/11ty/webc#aggregating-css-and-js

**Comment by mayank99** [comment]
Right, I saw `setBundlerMode` but I don't think it quite solves this problem.

Here's what I want:
1. aggregate CSS/JS for component instances
   - this is useful to prevent duplication and to allow extra bundling steps (e.g. externalizing to a file, applying postcss)
2. don't touch CSS/JS for the page that _uses_ those components
   - this makes sense because the page itself is not always a webc component (e.g. could be a page generated by another framework)

If I use `setBundlerMode(false)`, then it fixes 2 but breaks 1.

**Comment by mayank99** [comment]
For more context, I'm making a vite plugin for webc, and I'm currently working around this by [manually adding `webc:keep`](https://github.com/mayank99/vite-plugin-webc/blob/7881b7d2750dcb8bb333ff43edaad034f1b2499f/plugin/index.mjs#L34-L39) to all the unrelated scripts and styles on the page before handing them off to webc.

**Comment by zachleat** [comment]
Ah I know I haven’t really answered your question yet but I wonder if `webc:raw` would do what you wanted

**Comment by mayank99** [comment]
Ah yeah, `webc:raw` is probably a better fit for this use case. Still a workaround because I need to add it to non-webc related code, but it works 👍
